shared/kubernetes/provider.go

```diff
@@ -239,8 +239,8 @@ func (provider *Provider) GetMizuApiServerPodObject(opts *ApiServerOptions, moun
 			},
```

Comment:
Most changes in this file are due to `go fmt`, `CreateService` is the only func that has any real changes